src/fsharp/LexFilter.fs

```diff
@@ -179,6 +180,18 @@ let infixTokenLength token =
     | COLON_QMARK_GREATER -> 3
     | _ -> assert false; 1
 
+/// Matches against a left-parenthesis-like token that is valid in expressions.
+let (|TokenLExprParen|_|) =
+    function
+    | BEGIN | LPAREN | LBRACE _ | LBRACE_BAR | LBRACK | LBRACK_BAR | LQUOTE _ | LESS true
```

Comment:
What about `LBRACK_LESS`?

Reply:
Indentation warnings are done differently with `[<` and `>]` it seems, where adding them in this active pattern causes more offside warnings, while removing them doesn't add offside warnings in attributes

Reply:
OK thank you